new-version.sh

```diff
@@ -144,6 +144,7 @@ perl -i -pe"s/^version=.*/version=${RELEASE_VERSION}/g" ./integration/spark-exte
 perl -i -pe"s/^version=.*/version=${RELEASE_VERSION}/g" ./integration/flink/gradle.properties
 perl -i -pe"s/^version=.*/version=${RELEASE_VERSION}/g" ./integration/flink/examples/flink1-test-apps/gradle.properties
 perl -i -pe"s/^version=.*/version=${RELEASE_VERSION}/g" ./integration/flink/examples/flink2-test-apps/gradle.properties
+perl -i -pe"s/^version=.*/version=${RELEASE_VERSION}/g" ./integration/hive/gradle.properties
```

Comment:
Probably `integration/hive/hive-openlineage-hook/src/main/resources/io/openlineage/hive/client/version.properties` should be updated as well